Documentation/ApiOverview/Routing/Introduction.rst

```diff
@@ -20,28 +16,66 @@ When TYPO3 serves a request, it maps the incoming URL to a specific page or acti
++==========+============+==========+=====+======+======================+====================+================+================+=======+==================+================+================+========================+
+| Protocol | Subdomain  | Domain   | TLD | Port | Site Language Prefix | Slug               | Enhanced Route                                             |                |                |                        |
++----------+------------+----------+-----+------+----------------------+--------------------+-----------------------------------------+------------------+----------------+----------------+------------------------+
+|          | Hostname                    |      |                      |                    | Route Enhancer                          | Route Decorator  | Query string   | argument value | Location Hash / Anchor |
++----------+-----------------------------+------+----------------------+--------------------+-----------------------------------------+------------------+----------------+----------------+------------------------+
+|                                               |  Route / Permalink                                                                                     |                                                          |
++-----------------------------------------------+--------------------------------------------------------------------------------------------------------+----------------+----------------+------------------------+
+| URL (no arguments, unlike the URI)                                                                                                                                     |                |                |                        |
++--------------------------------------------------------------------------------------------------------------------------------------------------------+----------------+----------------+------------------------+
+| URI (everything)                                                                                                                                                                                                  |
++-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+..  hint::
+
+    Please note that the following terminology is based on technical terms used in the TYPO3 Core,
+    due to their Class/Object and Interface names.
+
+..  index:: Routing; Route
 
 Route
-   The "speaking URL" as a whole (without the domain part); for example `/news/detail/2019-software-update`
+   The "speaking URL" as a whole (without the domain parts); for example `/en/about-us/our-team/john-doe/publications/index.xhtml`.
+   This is also sometimes referred to as `permalink`, some definitions also include the `Query string`
```

Comment:
The query string is everything behind `?`. As the query path does not contain any `?` there is no query string in your example.

Reply:
In the table there is a query string; it is not part of a "speaking URL" by default, so the URL example before it doesn't contain it - it is explained later on so I think this shouldn't be a problem.